ckanext/opendata_theme/opengov_custom_footer/templates/footer.html

```diff
@@ -0,0 +1,18 @@
+<footer class="site-footer">
+  <div class="container">
+    {% block footer_content %}
+    {% set layout_type = h.get_footer_data('layout_type')|int %}
+    <div class="row">
+        {% for index in range(layout_type) %}
```

Comment:
For the default layout the default content should be displayed, but the template here is good enough for a POC.